projects/unbound/build.sh

```diff
@@ -46,4 +50,88 @@ $CXX $CXXFLAGS -std=c++11 \
   libworker.o context.o \
   $LIBOBJS
 
+$CXX $CXXFLAGS -std=c++11 \
```

Comment:
This is fine for now, but we recommend integrating these into the upstream target if possible.